lib/compilers/php.js

```diff
@@ -0,0 +1,56 @@
+var Compile = require('../base-compiler'),
+	fs = require('fs-extra'),
+	php = require('../php'),
+	utils = require('../utils');
+
+function compilePHP(info, env) {
+    var compiler = new Compile(info, env);
+    compiler.supportsIntel = false;
+    compiler.couldSupportASTDump = function() { return false; };
```

Comment:
Note that server-side code supports ES6 notations, which would remove most of the boilerplate in lines like this. For this example: `...couldSupportASTDump = () => false;`